mlflow/ml-package-versions.yml

```diff
@@ -242,6 +242,17 @@ spacy:
     run: |
       pytest tests/spacy/test_spacy_model_export.py --large
 
+spark:
+  package_info:
+    pip_release: "pyspark"
+
+  autologging:
+    minimum: "3.0.0"
+    maximum: "3.1.1"
```

Comment:
This disables pyspark autologging in pyspark 2.4.x. Do we still need to support it or not?